packages/runtime-core/src/helpers/resolveAssets.ts

```diff
@@ -26,7 +26,7 @@ export function resolveComponent(
   return resolveAsset(COMPONENTS, name, true, maybeSelfReference) || name
 }
 
-export const NULL_DYNAMIC_COMPONENT = Symbol()
+export const NULL_DYNAMIC_COMPONENT = Symbol(__DEV__ ? 'Null' : undefined)
```

Comment:
I did this to make it Similar to `Text`, `Static`, and other existing symbols in vnode.ts. It could also be moved with the other existing vnodes